animations-tutorial.md

```diff
@@ -0,0 +1,589 @@
+[`addStatusListener`](http://docs.flutter.io/flutter/animation/Animation/addStatusListener.html).
+A Listener is called whenever the value of the animation changes. The most common behavior of a Listener is to call setState() to cause a rebuild. A StatusListener is called when an animation begins, ends, moves forward, or moves reverse, as defined by [`AnimationStatus`](http://docs.flutter.io/flutter/animation/AnimationStatus-class.html).
+
+We’ll use `addListener` in the next section and we’ll use `addStatusListener` in Monitoring the progress of the animation.
+
+## Rendering animations
+
+So far all we’ve done is to generate a sequence of numbers over time. You’ll notice we haven’t actually rendered anything.
+
+To render with an `Animation<>` object, store the `Animation<>` object as a member of your `Widget`, then use its value to decide how to draw.
+
+Consider this application that draws the Flutter logo without animation:
+
+```dart
+import 'package:flutter/material.dart';
+
+const kLogoUrl = 'https://raw.githubusercontent.com/dart-lang/logos/master/logos_and_wordmarks/dart-logo.png';
+
+class LogoApp extends StatefulComponent {
+  LogoAppState createState() => new LogoAppState();
+}
+
+class LogoAppState extends State<LogoApp> {
+
+  Widget build(BuildContext context) {
+    return new Center(
+        child: new Container(
+            margin: new EdgeDims.symmetric(vertical: 10.0),
+            height: 300.0,
+            width: 300.0,
+            child: new NetworkImage(src: kLogoUrl)));
+  }
+}
+
+void main() {
+  runApp(new LogoApp());
+}
+```
+
+This listing shows the changes necessary to animate the logo to grow outward: (Changes noted in bold)
+
+<pre><code class="language-dart">
+<b>import 'package:flutter/animation.dart';</b>
+import 'package:flutter/material.dart';
+
+const kLogoUrl = "https://raw.githubusercontent.com/dart-lang/logos/master/logos_and_wordmarks/dart-logo.png";
+
+class LogoApp extends StatefulComponent {
+  LogoAppState createState() => new LogoAppState();
+}
+
+class LogoAppState extends State&lt;LogoApp&gt; {
+  Animation&lt;double&gt; animation;
+<b>
+  initState() {
+    super.initState();
+    AnimationController controller =
+        new AnimationController(duration: const Duration(milliseconds: 2000));
+    animation = new Tween(begin: 0.0, end: 300.0).animate(controller)
+      ..addListener(() {
+        setState(() {
+          // the state that has changed here is the animation object’s value
+        });
+      });
+    controller.forward();
+  }
+</b>
+  Widget build(BuildContext context) {
+    return new Center(
+        child: new Container(
+            margin: new EdgeDims.symmetric(vertical: 10.0),
+            height: <b>animation.value</b>,
+            width: <b>animation.value</b>,
+            child: new NetworkImage(src: kLogoUrl)));
+  }
+}
+
+void main() {
+  runApp(new LogoApp());
+}
+</code></pre>
+
```

Comment:
Starting from here the rendering done (visible at https://github.com/flutter/website/blob/animation-doc/animations-tutorial.md) is without markdown evaluation. Seems to be due to the use of `<code></code>`.